librz/include/rz_analysis.h

```diff
@@ -78,18 +77,18 @@ enum {
 };
 
 // used from core/analysis.c
-#define RZ_ANALYSIS_ADDR_TYPE_EXEC      1
-#define RZ_ANALYSIS_ADDR_TYPE_READ      1 << 1
-#define RZ_ANALYSIS_ADDR_TYPE_WRITE     1 << 2
-#define RZ_ANALYSIS_ADDR_TYPE_FLAG      1 << 3
-#define RZ_ANALYSIS_ADDR_TYPE_FUNC      1 << 4
-#define RZ_ANALYSIS_ADDR_TYPE_HEAP      1 << 5
-#define RZ_ANALYSIS_ADDR_TYPE_STACK     1 << 6
-#define RZ_ANALYSIS_ADDR_TYPE_REG       1 << 7
-#define RZ_ANALYSIS_ADDR_TYPE_PROGRAM   1 << 8
-#define RZ_ANALYSIS_ADDR_TYPE_LIBRARY   1 << 9
-#define RZ_ANALYSIS_ADDR_TYPE_ASCII     1 << 10
-#define RZ_ANALYSIS_ADDR_TYPE_SEQUENCE  1 << 11
+#define RZ_ANALYSIS_ADDR_TYPE_EXEC 1
+#define RZ_ANALYSIS_ADDR_TYPE_READ 1 << 1
+#define RZ_ANALYSIS_ADDR_TYPE_WRITE 1 << 2
+#define RZ_ANALYSIS_ADDR_TYPE_FLAG 1 << 3
+#define RZ_ANALYSIS_ADDR_TYPE_FUNC 1 << 4
+#define RZ_ANALYSIS_ADDR_TYPE_HEAP 1 << 5
+#define RZ_ANALYSIS_ADDR_TYPE_STACK 1 << 6
+#define RZ_ANALYSIS_ADDR_TYPE_REG 1 << 7
+#define RZ_ANALYSIS_ADDR_TYPE_PROGRAM 1 << 8
+#define RZ_ANALYSIS_ADDR_TYPE_LIBRARY 1 << 9
+#define RZ_ANALYSIS_ADDR_TYPE_ASCII 1 << 10
+#define RZ_ANALYSIS_ADDR_TYPE_SEQUENCE 1 << 11
```

Comment:
Can aligning indents like here be preserved? The change decreases readability.

Reply:
I added "AlignConsecutiveMacros" but it doesn't seem to work...

Reply:
Setting `UseTab` to `UT_ForContinuationAndIndentation` or `UT_AlignWithSpaces` instead of `Always` fixes it by replacing all non indent/continuation tabs with spaces and preventing weird tab behavior in macro alignment. There are other cases of this issue because tabs were mixed with spaces.